.vsts-ci/misc-analysis.yml

```diff
@@ -60,7 +58,7 @@ phases:
       Write-Host "##vso[results.publish type=NUnit;mergeResults=true;runTitle=Common Tests;publishRunAttachments=true;resultFiles=$path;]"
       if($results.TotalCount -eq 0 -or $results.FailedCount -gt 0)
       {
-        throw "markdown link tests failed"
+        throw "Markdown tests failed"
```

Comment:
Revert style.

Reply:
same here, now this is both links and spelling (not style)